public/data/chapters/en.json

```diff
@@ -1,686 +1,800 @@
 {
     "1": {
-        "translatedName": "The Opener",
+        "revelationPlace": "makkah",
         "transliteratedName": "Al-Fatihah",
         "versesCount": 7,
-        "revelationPlace": "makkah"
+        "translatedName": "The Opener",
+        "slug": "al-fatihah"
     },
     "2": {
-        "translatedName": "The Cow",
+        "revelationPlace": "madinah",
         "transliteratedName": "Al-Baqarah",
         "versesCount": 286,
-        "revelationPlace": "madinah"
+        "translatedName": "The Cow",
+        "slug": "al-baqarah"
     },
     "3": {
-        "translatedName": "Family of Imran",
+        "revelationPlace": "madinah",
         "transliteratedName": "Ali 'Imran",
         "versesCount": 200,
-        "revelationPlace": "madinah"
+        "translatedName": "Family of Imran",
+        "slug": "ali-imran"
     },
     "4": {
-        "translatedName": "The Women",
+        "revelationPlace": "madinah",
         "transliteratedName": "An-Nisa",
         "versesCount": 176,
-        "revelationPlace": "madinah"
+        "translatedName": "The Women",
+        "slug": "4th-surah"
     },
     "5": {
-        "translatedName": "The Table Spread",
+        "revelationPlace": "madinah",
         "transliteratedName": "Al-Ma'idah",
         "versesCount": 120,
-        "revelationPlace": "madinah"
+        "translatedName": "The Table Spread",
+        "slug": "5th-surah"
     },
     "6": {
-        "translatedName": "The Cattle",
+        "revelationPlace": "makkah",
         "transliteratedName": "Al-An'am",
         "versesCount": 165,
-        "revelationPlace": "makkah"
+        "translatedName": "The Cattle",
+        "slug": "quran-6th-surah"
     },
     "7": {
-        "translatedName": "The Heights",
+        "revelationPlace": "makkah",
         "transliteratedName": "Al-A'raf",
         "versesCount": 206,
-        "revelationPlace": "makkah"
+        "translatedName": "The Heights",
+        "slug": "surah-al-araf"
```

Comment:
also this one. Al'anam has `6th` in it, Al maidah has 5th but at tawbah does not.

Reply:
The data is fetched from BE.